website/public/css/global-colors.styl

```diff
@@ -19,6 +19,7 @@ $color-mp = $best
 $color-toolbar = lighten($color-herobox, 70%)
 $color-options-menu = lighten($color-herobox, 85%)
 $color-options-submenu = lighten($color-herobox, 75%)
+$color-options-blurb = lighten($color-herobox, 75%)
```

Comment:
Since this is identical to `$color-options-submenu`, I don't think it is necessary; you could just use `$color-options-submenu`. But wait for an admin (most likely @crookedneighbor or @Alys) to share their opinions before changing anything.